### DISCUSSION TITLE
Antenna TV m3u8

### DISCUSSION THREAD

**Discussion by RickyE23** [iptv-org/community#418] [Q&A] [answered]
Looking for another M3U8 link for Antenna TV I can add to Stremium...Can anyone help please....It seems the one I had just ceased working,along with the Rewind TV link which I didn't watch frequently enough to add back...Thanks in advance !

**Comment by Toei79** [comment]
question you don't have it on antenna?
idk whats streamium but you can make stream link from antenna ota signals. with the right device. no only hdhomerun.

**Reply by RickyE23** [reply]
No it's not OTA locally... Stremium is an app with about 80 existing channels where users can add M3U8 links...I've added MeTV,Heroes,Rewind TV but I haven't found functioning links for Antenna TV or Decades.

**Reply by Toei79** [reply]
sounds cool, i have to check. 
i see. i give idea to you if, idk how good you are on it , but using tvheadend in one market where you get antena tv you can streaming it,  
i go to my job and use a tvheadend app and i can see my home antenna channels, just idea, if you have a friend in one market that can helps, you need a pi, usb tuner and probably someone with fiber, its maybe a long shot and just idea.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel
